dashboard/src/containers/AppViewContainer/AppViewContainer.tsx

```diff
@@ -30,6 +31,13 @@ function mapDispatchToProps(dispatch: ThunkDispatch<IStoreState, null, Action>)
     deleteApp: (releaseName: string, ns: string, purge: boolean) =>
       dispatch(actions.apps.deleteApp(releaseName, ns, purge)),
     getApp: (releaseName: string, ns: string) => dispatch(actions.apps.getApp(releaseName, ns)),
+    getResource: (
```

Comment:
I'd have preferred to have this method, or even a specific one (see below) in the SecretsTableContainer instead.

```
# With specific method I meant this one in the SecretsTableContainer
getSecret(namespace, name)
```

Reply:
BTW, for some reason I can not find where this method is being used.

Reply:
I forgot to remove this. You are right, it's used as `getSecret` in the SecretTableContainer